internal/provider/resource_ably_queue_test.go

```diff
@@ -0,0 +1,83 @@
+package ably_control
+
+import (
+	"fmt"
+	"testing"
+
+	ably_control_go "github.com/ably/ably-control-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAblyQueue(t *testing.T) {
+	app_name := acctest.RandStringFromCharSet(15, acctest.CharSetAlphaNum)
+	queue_name := acctest.RandStringFromCharSet(15, acctest.CharSetAlphaNum)
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAblyQueueConfig(app_name, ably_control_go.NewQueue{
+					Name:      queue_name,
+					Ttl:       44,
+					MaxLength: 83,
+					Region:    ably_control_go.EuWest1A,
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ably_app.app0", "name", app_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "name", queue_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "ttl", "44"),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "max_length", "83"),
+				),
+			},
+			{
+
+				Config: testAccAblyQueueConfig(app_name, ably_control_go.NewQueue{
+					Name:      queue_name + "new",
+					Ttl:       99,
+					MaxLength: 83,
+					Region:    ably_control_go.UsEast1A,
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ably_app.app0", "name", app_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "name", queue_name+"new"),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "ttl", "99"),
```

Comment:
Ran the Acceptance test. Got an error stating that TTL should be => 60. 
`
=== RUN   TestAccAblyQueue
    resource_ably_queue_test.go:15: Step 2/2 error: Error running apply: exit status 1
        
        Error: Error creating Resource
        
          with ably_queue.queue0,
          on terraform_plugin_test.tf line 19, in resource "ably_queue" "queue0":
          19: resource "ably_queue" "queue0" {
        
        Could not create resource, unexpected error: /apps/IPGsPg/queues: Invalid
        resource: code 40022: status code: 422 see: https://help.ably.io/error/40022
          ttl:
            must be less than or equal to 60
`